src/app-gocardless/bank-factory.js

```diff
@@ -47,6 +48,7 @@ export const banks = [
   NorwegianXxNorwnok1,
   SEBKortBankAB,
   SEBPrivat,
+  RevolutRevolt21,
```

Comment:
Could this be moved above the SEB banks to keep the list in alphabetical order please?